core/android_validation_constants.py

```diff
@@ -34,11 +35,19 @@
     'AlgebraicExpressionInput', 'Continue', 'DragAndDropSortInput',
     'EndExploration', 'FractionInput', 'ImageClickInput', 'ItemSelectionInput',
     'MathEquationInput', 'MultipleChoiceInput', 'NumericExpressionInput',
-    'NumericInput', 'NumberWithUnits', 'RatioExpressionInput', 'TextInput'
+    'NumericInput', 'RatioExpressionInput', 'TextInput'
 ]
 
 # This is linked to SUPPORTED_CONTENT_LANGUAGES_FOR_ANDROID in constants.ts.
-SUPPORTED_LANGUAGES = ['en']
+SUPPORTED_LANGUAGE_CODES = ['en', 'hi-en', 'ar', 'hi', 'pt-br']
```

Comment:
Why is this redefined? Cannot we use `constant.SUPPORTED_CONTENT_LANGUAGES_FOR_ANDROID`? Ditto for some other constants in this file.

Reply:
@BenHenning Will maybe remember the reason.